src/styles/stars.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: Arial, sans-serif;
+  box-sizing: border-box;
+}
```

Comment:
Avoid using the `*` selector as it impacts performance. It's better to set styles only for elements that require them.